gems/pending/appliance_console.rb

```diff
@@ -28,7 +28,8 @@
 include HighLine::SystemExtensions
 
 require 'i18n'
-LOCALES = File.join(File.expand_path(File.dirname(__FILE__)), "appliance_console/locales/*.yml")
+locales_dir = ENV['CONTAINER'].to_s.casecmp("true") == 0 ? "container" : "appliance"
```

Comment:
yeah, nobody does that...the values are either set or not set, and people pick different values, like what @carbonin said, so typically we just look for presence of the option.